doc/contributing.rst

```diff
@@ -181,17 +181,17 @@ The two examples below might help make this process easier to understand:
 
      2. Run ``python tests/update_files.py --tests tests --outputs test_outputs`` to update test outputs. The files affected by your change are shown as added/deleted. You also see the following warning: 
 
-        .. code-block:: 
+        .. code-block:: text
      
             WARNING: X input files for rsmcompare/rsmsummarize tests have been updated. You need to re-run these tests and update test outputs
 
      3. This means that the changes you made to the code changed the outputs for one or more ``rsmtool``/``rsmeval`` tests that served as inputs to one or more ``rsmcompare``/``rsmsummarize`` tests. Therefore, it is likely that the current test outputs no longer match the expected output and the tests for those two tools must be be re-run.
 
      4. Run ``nosetests --nologcapture tests/*rsmsummarize*.py`` and ``nosetests --nologcapture tests/*rsmcompare*.py``. If you see any failures, make sure they are related to the changes you made since those are expected.
 
-     3. Next, re-run ``python tests/update_files.py --tests tests --outputs test_outputs`` which should only update the outputs for the ``rsmcompare``/``rsmsummarize`` tests. 
+     5. Next, re-run ``python tests/update_files.py --tests tests --outputs test_outputs`` which should only update the outputs for the ``rsmcompare``/``rsmsummarize`` tests. 
 
-     4. If this is the case, you are now ready to commit your changes. 
+     6. If this is the case, you are now ready to commit your changes. 
```

Comment:
Looks good! One thing we could do to get around having to manually number the steps is use `#.` in place of each number. This will make turn into number automatically.